src/common/command_line.cpp

```diff
@@ -36,6 +36,14 @@
 #include "cryptonote_config.h"
 #include "string_tools.h"
 
+#ifdef WIN32
```

Comment:
Please use _WIN32 (apparently WIN32 is only defined if you've got the right include included already, so it's error prone)

Reply:
What you are thinking is MSVC defines (not that the code compiles there anyway), but sure.

Reply:
No idea about windows. I'm repeating what I was told by someone who does know, but maybe I'm assuming things. If you think that is wrong, explain why.

Reply:
To put it in simple terms:

gcc defines `WIN32` and `_WIN32`
msvc defines `_WIN32`

You use `WIN32` (and I did out of laziness), however you suggestion is better practice.